modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -374,8 +374,7 @@ private void waitForScale(IgniteImpl node, int targetDataNodesCount) throws Inte
         assertTrue(IgniteTestUtils.waitForCondition(() -> {
             long causalityToken = node.metaStorageManager().appliedRevision();
 
-            long msSafeTime = node.metaStorageManager().timestampByRevision(causalityToken).longValue();
-            int catalogVersion = node.catalogManager().activeCatalogVersion(msSafeTime);
+            int catalogVersion = node.catalogManager().latestCatalogVersion();
```

Comment:
Why have made this change?

Reply:
It was an incorrect test because it capturedd not final catalog state.

Reply:
Agreed, ms idle safe time is not taken into account while retrieving msSafeTime, my code was buggy